Automattic/simplenote-android
# Update Welcome Note

### Fix
Remove the "blue" reference from the welcome note since the **_New Note_** floating action button may no longer be blue if a style other than **_Default_** or **_Classic_** is selected.

### Test
0. Clear app data.
1. Log in to app.
2. Tap welcome note in list.
3. Notice "tap the "New Note" button" no longer has "blue" reference.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
